README.md

```diff
@@ -0,0 +1,66 @@
+# Analysis Schema
+
+A declarative schema and workflow framework for analysis and visualization of physical systems.
+
+The Analysis Schema creates a workflow system that processes a configuration file where users specify _what_ they would like the code to do, instead of _how_ to do it to complete analysis and visualization tasks. It supports high level analysis description that can be saved, shared, and reused in JSON document. 
+
+It consists of three pieces:
+
+1. A structured schema that guides the workflow. See the `yt_analysis_schema.json`. 
+2. An engine that connects input from the configuration file to the underlying code.
+3. The configuration file. 
+
```

Comment:
We should add an Installation section here. That can wait for a followup PR though...

Reply:
I agree, I can add this once we have the packaging done.